e2e/tests/transfer/forwarding_test.go

```diff
@@ -95,3 +99,69 @@ func (s *TransferForwardingTestSuite) TestThreeChainSetup() {
 		s.Require().Equal(expected, actualBalance.Int64())
 	})
 }
+
+func (s *TransferForwardingTestSuite) TestForwarding_WithLastChainBeingICS20v1_Succeeds() {
+	ctx := context.TODO()
+	t := s.T()
+
+	relayer, chains := s.GetRelayer(), s.GetAllChains()
+
+	chainA, chainB, chainC := chains[0], chains[1], chains[2]
+
+	channelAtoB := s.GetChainAChannel()
+
+	// Creating a new path between chain B and chain C with a ICS20-v1 channel
```

Comment:
I went a bit back and forth on how to do this and in the end I thought the cleanest way was to just create a new path for this test because the alternative would be to move this to a separate TestSuite which I didn't like very much.